crates/client/data-availability/src/lib.rs

```diff
@@ -63,6 +64,16 @@ pub enum DaMode {
     Sovereign,
 }
 
+impl Display for DaMode {
```

Comment:
we can probably just use StringEnum from strum/serde ?

Reply:
What is StringEnum?